openweathermap-ios/openweathermap-iosTests/BookmarkLocation/Manager/BookmarkLocationManagerTest.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  BookmarkLocationServiceTest.swift
+//  openweathermap-ios
+//
+//  Created by Cassio Sousa on 24/04/20.
+//
+
+import XCTest
+
+class BookmarkLocationManagerTest: XCTestCase {
+
+    private var bookmarkLocationManager = BookmarkLocationManager()
+    
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    /**
+        It validates if not exits elementes
+     */
+    func testNoElementsBookmarkLocation() throws {
+        UserDefaults.standard.removeObject(forKey: BookmarLocationEnumaration.bookmarkLocation.rawValue)
+        
+        let results = bookmarkLocationManager.getAll();
+        XCTAssert(results.count == 0)
+    }
+    
+    /**
+        It makes test to create one new Bookmark location
+     */
+    func testCcreateOneBookmarkLocation() throws {
+        UserDefaults.standard.removeObject(forKey: BookmarLocationEnumaration.bookmarkLocation.rawValue)
+        let bookmarkLocation = BookmarkLocation(title: "Florida - USA", latitude: "27.607668", longitude: "-81.604064")
```

Comment:
Check all places you are declaring a `bookmarkLocation`. Prefer to use only `bookmark`. It's shorter, and its cleaner.

Pay attention, I am not asking you to change the class name. Only the var/let declarations. See the suggestion below:
```suggestion
        let bookmark = BookmarkLocation(title: "Florida - USA", latitude: "27.607668", longitude: "-81.604064")
```

Reply:
Fixed in b6bb083